jenkins/cross-cluster-replication/perf-test.jenkinsfile

```diff
@@ -0,0 +1,139 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
```

Comment:
Do we  plan to add regression tests for this jenkins file? Similar to https://github.com/opensearch-project/opensearch-build/tree/main/tests/jenkins/jenkinsjob-regression-files
More details on how to add: https://github.com/opensearch-project/opensearch-build/blob/main/DEVELOPER_GUIDE.md#regression-tests

Reply:
Added the test for the jenkins file